archeo/style.css

```diff
@@ -125,7 +125,6 @@ body > .is-root-container,
 .wp-site-blocks > .wp-block-cover,
 .wp-site-blocks > .wp-block-template-part > .wp-block-group.has-background,
 .wp-site-blocks > .wp-block-template-part > .wp-block-cover,
-body > .is-root-container > .wp-block-group.has-background,
 body > .is-root-container > .wp-block-cover,
 body
```

Comment:
This formatting is really weird — not sure why?

Reply:
no I'm not sure what's happened there...